tests/conftest.py

```diff
@@ -32,13 +32,25 @@
 
 
 def pytest_collection_modifyitems(config, items):
-    excluded_tests = []
+    def get_excluded_tests(test_exclude_file):
+        if os.path.exists(test_exclude_file):
+            excluded_tests = []
+            with open(test_exclude_file) as skip_names_file:
+                excluded_tests = skip_names_file.readlines()
+            return excluded_tests
+
     # global skip file
     test_path = os.path.split(__file__)[0]
     test_exclude_file = os.path.join(test_path, 'skipped_tests.tbl')
-    if os.path.exists(test_exclude_file):
-        with open(test_exclude_file) as skip_names_file:
-            excluded_tests = skip_names_file.readlines()
+
+    # additional skip file, where gpu queue not supported
```

Comment:
it should not be additional file. It should be standalone files with all tests disabled for GPU.